src/app/(platform)/foro/categoria/[id]/page.tsx

```diff
@@ -0,0 +1,282 @@
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { Badge } from '@/components/ui/badge';
+import { Button } from '@/components/ui/button';
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Input } from '@/components/ui/input';
+import { Heading2 } from '@/components/ui/heading-2';
+import { ArrowLeft, Search, Pin, Heart, MessageCircle, Filter, SortDesc, Plus } from 'lucide-react';
+import Link from 'next/link';
+import prisma from '@/lib/prisma';
+import { formatDistanceToNow } from 'date-fns';
+import { es } from 'date-fns/locale';
+import { notFound } from 'next/navigation';
+
+const CategoryPage = async ({ params }: { params: { id: string } }) => {
+  const category = await prisma.postCategory.findUnique({
+    where: { slug: params.id },
+    include: {
+      _count: {
+        select: { posts: true },
+      },
+    },
+  });
+
+  if (!category) {
+    notFound();
+  }
+
+  const posts = await prisma.post.findMany({
+    where: { categoryId: category.id },
+    orderBy: [{ isPinned: 'desc' }, { createdAt: 'desc' }],
+    include: {
+      author: true,
+      _count: {
+        select: {
+          comments: true,
+          likes: true,
+        },
+      },
+    },
+  });
+
+  // Estadísticas de la categoría
+  const totalPosts = posts.length;
+  const totalComments = await prisma.postComment.count({
+    where: {
+      post: {
+        categoryId: category.id,
+      },
+    },
+  });
+  return (
```

Comment:
Acá estaría bueno agregar un par de líneas en blanco
```suggestion
  const totalPosts = posts.length;
  
  const totalComments = await prisma.postComment.count({
    where: {
      post: {
        categoryId: category.id,
      },
    },
  });
  
  return (
```